src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/Api/IUnitTestingSolutionCrawlerRegistrationServiceAccessor.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
+{
+    internal interface IUnitTestingSolutionCrawlerRegistrationServiceAccessor : IWorkspaceService
```

Comment:
Not sure if we want to expose `IWorkspaceService` through this interface. Any reason this needs to derive from `IWorkspaceService`?

Reply:
I am following the third sample in this doc [here](https://github.com/dotnet/roslyn/blob/260772bbdf6e515dce1f327cfbb74208ebf9892a/src/Tools/ExternalAccess/Patterns.md#accessing-workspace-services)
```
[ExternalApi]
internal interface IPartnerServiceNameAccessor : IWorkspaceService {
    // Define members for accessing IServiceName functionality
}
```
Is this pattern out-of-date?

Reply:
The pattern is correct but you don't need `[ExternalApi]`.